acceptance/gossip_peerings_test.go

```diff
@@ -83,6 +90,7 @@ func checkGossipPeerings(t *testing.T, l *localcluster.Cluster, attempts int, do
 				fmt.Printf("... all nodes verified in the cluster\n")
 				return
 			}
+			time.Sleep(1 * time.Second)
```

Comment:
moved to the end of the loop to give better visibility into working "kill/restart" tests. The nodes re-join gossip very quickly and we could not see it.